### ISSUE TITLE
Tsum v63 auto play hang

### ISSUE THREAD

**Issue by Sean-Hsueh** [Hellomon/rerouter#24]
We received a player saying v63 hang during auto play game, and he provided Youtube link showing the issue:

https://www.youtube.com/watch?v=UGHRpJt2mBY
https://www.youtube.com/watch?v=BjRqUXXxmIs

Not sure if is caused by page matching blocked by the ads (that ads banner had been removed in newer Robotmon release). @mcs can you please confirm that is the case? 

(Part of the video are in Chinese so feel free to let us know if any translation is needed)

**Comment by mcs** [comment]
Oh oh, I think you are right mentioning the ad banner. I try to match the position with the script's gameplay detection.

**Comment by mcs** [comment]
I have to revise my previous post: the ad banner is probably not the cause of the problem. I adjusted some page color definitions to quite "real" values, but the lowest resolution I used for testing was 540x960, while the uploader uses 480x800 (which can be seen at timestamp [1:09](https://youtu.be/UGHRpJt2mBY?si=xarhjk5lMnVEYBGP&t=69). I already noticed that the TsumBeta script compresses its screenshots more or less independently of the real used screen resolution which leads (depending on the area and other settings) to ingame screenshots with resolutions of just around 100 px per side.

I think I'll review the code used for CookieKingdom to see if that behaves different (in terms of "more smart" with respect to the real device's screen resolution). Until then and to provide a quick fix, I'll revert the related color adjustments and prepare a v64 bugfix release.
